docs/sidebars.js

```diff
@@ -485,115 +485,182 @@ module.exports = {
-          label: "Organization Objects",
-          link: {
-            type: "generated-index",
-            title: "Management API",
-            slug: "/apis/resources/mgmt",
-            description:
-              "The management API is as the name states the interface where systems can mutate IAM objects like, organizations, projects, clients, users and so on if they have the necessary access rights. To identify the current organization you can send a header x-zitadel-orgid or if no header is set, the organization of the authenticated user is set.",
-          },
-          items: require("./docs/apis/resources/mgmt/sidebar.js"),
-        },
-        {
-          type: "category",
-          label: "Instance Objects",
-          link: {
-            type: "generated-index",
-            title: "Admin API",
-            slug: "/apis/resources/admin",
-            description:
-              "This API is intended to configure and manage one ZITADEL instance itself.",
-          },
-          items: require("./docs/apis/resources/admin/sidebar.js"),
-        },
-        {
-          type: "category",
-          label: "Instance Lifecycle",
-          link: {
-            type: "generated-index",
-            title: "System API",
-            slug: "/apis/resources/system",
-            description:
-              "This API is intended to manage the different ZITADEL instances within the system.\n" +
-              "\n" +
-              "Checkout the guide how to access the ZITADEL System API.",
-          },
-          items: require("./docs/apis/resources/system/sidebar.js"),
-        },
-        {
-          type: "category",
-          label: "User Lifecycle (Beta)",
-          link: {
-            type: "generated-index",
-            title: "User Service API (Beta)",
-            slug: "/apis/resources/user_service",
-            description:
-              "This API is intended to manage users in a ZITADEL instance.\n" +
-              "\n" +
-              "This project is in beta state. It can AND will continue breaking until the services provide the same functionality as the current login.",
-          },
-          items: require("./docs/apis/resources/user_service/sidebar.js"),
-        },
-        {
-          type: "category",
-          label: "Session Lifecycle (Beta)",
+          label: "Service Based (V1)",
+          collapsed: false,
           link: {
             type: "generated-index",
-            title: "Session Service API (Beta)",
-            slug: "/apis/resources/session_service",
-            description:
-              "This API is intended to manage sessions in a ZITADEL instance.\n" +
-              "\n" +
-              "This project is in beta state. It can AND will continue breaking until the services provide the same functionality as the current login.",
+            title: "Service Based APIs (V1)",
+            slug: "/apis/services/",
+            description: "Service based API definitions",
```

Comment:
I think we should add a more descriptive description here.
What does service based API mean?
When should I use this one?

Maybe we can even link to a more detailed page, where we describe when to use what.